spec/ahoy/intercom_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "spec_helper"
+
+RSpec.describe Ahoy::Intercom do
+
+  before do
+    ahoy.authenticate(user)
+  end
+
+  after { intercom.users.delete(intercom_user) }
+
+  let!(:ahoy) { Ahoy::Tracker.new }
+  let(:user_info) { {email: 'test_user@example.com', name: 'Test User'} }
+  let!(:user) { User.create(user_info) }
+  let!(:intercom) { ::Intercom::Client.new(token: ENV['INTERCOM_ACCESS_TOKEN']) }
+  let!(:intercom_user) { intercom.users.create(user_info) }
+
+  it "stored into database" do
```

Comment:
it doesnt need to store anything in database, this gem is an Intercom store, nothing else.